cwms-data-api/src/main/java/cwms/cda/api/LevelsController.java

```diff
@@ -220,8 +220,8 @@ public void delete(@NotNull Context ctx, @NotNull String levelId) {
                         + "specified or default units above the NGVD-29 datum."),
                 @OpenApiParam(name = BEGIN, description = "Specifies the start of the time "
                         + "window for data to be included in the response. If this field is "
-                        + "not specified, any required time window begins 24 hours prior to "
-                        + "the specified or default end time."),
+                        + "not specified, any required time window begins 1/1/1900 at "
```

Comment:
This date doesn't match up with what is actually used.

Reply:
I would think leaving off the beginning or ending empty would just remove them from the select query bounds rather filling in a default.

Reply:
I updated the description to better match actual behavior. I also changed the default values to null, which causes the DAO to remove them from the query.